components/logicbutton/index.css

```diff
@@ -14,100 +14,124 @@ governing permissions and limitations under the License.
 @import "../vars/css/scales/spectrum-medium.css";
 @import "../commons/basebutton.css";
```

Comment:
I don't have a lot of context here. Does this mean that the LogicButton basebutton import needs to be the same as the one in the Button component? So instead of this it would be `@import "../commons/basebutton-coretokens.css";`? I tried that locally and everything looks the same visually, let me know if that's what you're thinking and I'll push up a change

Reply:
I'm not 100% on that one as I'm not too familiar with how we're using the `commons` imports; some questions related to that just came up for me when reviewing Overlay. I'd double-check with @pfulton , but considering `button` has been migrated, I think what you are suggesting would be correct.

Reply:
Did some digging and it looks like the main difference in `basebutton-coretokens.css` is that it includes mods, so I change it to that one and removed both the `vars` imports.